main.tf

```diff
@@ -161,7 +160,7 @@ module "eks" {
   update_kubeconfig_extra_args = var.update_kubeconfig_extra_args
   eks_master_role_names        = var.eks_master_role_names
   ssh_pvt_key_path             = local.ssh_pvt_key_path
-  bastion_user                 = local.bastion_user
+  bastion_user                 = try(var.bastion.username, null)
```

Comment:
Where we're still making this default to `ec2-user` everywhere else, is there any reason we don't do it here?